etn-ccis/blui-react-workflows
# Feature/4492 add custom success screen to registration workflow

Fixes # BLUI-4492



#### Changes proposed in this Pull Request:

- Added successScreen prop to registration workflow component to support custom success screen



#### Screenshots / Screen Recording (if applicable)

- 



#### To Test:

- yarn start:example3
- Edit AppRouter.tsx line 72 to pass `successScreen` prop to RegistrationWorkflow
- Click on Create Account button 
- Finish the steps for create account to see custom success screen we passed as a prop



#### Any specific feedback you are looking for?

- Please merge this PR once #359 gets merged.
